storage/include/libstqe.sh

```diff
@@ -17,8 +17,7 @@
 # Boston, MA 02110-1301, USA.
 #
 
-source /usr/share/beakerlib/beakerlib.sh
-source $(dirname $(readlink -f $BASH_SOURCE))/libbkrm.sh
+source $(dirname $(readlink -f $BASH_SOURCE))/../../cki_lib/libcki.sh
```

Comment:
Out of curiosity: what's the difference between this and simply the following?
```
source ../../cki_lib/libcki.sh
```

Reply:
`$(dirname $(readlink -f $BASH_SOURCE))/../../cki_lib/libcki.sh` is the absolute path, and `../../cki_lib/libcki.sh` is the relative path. It fails to load cki/libcki.sh if we use relative path. e.g.

- http://pastebin.test.redhat.com/799522

Hence, we have to use abolute path when sourcing shell script files.